abbreviation-plugin/part-1/package.json

```diff
@@ -1,40 +1,21 @@
 {
-  "dependencies": {
-    "@ckeditor/ckeditor5-basic-styles": "^38.1.1",
-    "@ckeditor/ckeditor5-core": "^38.1.1",
-    "@ckeditor/ckeditor5-dev-utils": "^35.0.3",
-    "@ckeditor/ckeditor5-editor-classic": "^38.1.1",
-    "@ckeditor/ckeditor5-essentials": "^38.1.1",
-    "@ckeditor/ckeditor5-heading": "^38.1.1",
-    "@ckeditor/ckeditor5-list": "^38.1.1",
-    "@ckeditor/ckeditor5-paragraph": "^38.1.1",
-    "@ckeditor/ckeditor5-theme-lark": "^38.1.1",
-    "@ckeditor/ckeditor5-ui": "^38.1.1",
-    "css-loader": "^5.2.7",
-    "postcss-loader": "^4.3.0",
-    "raw-loader": "^4.0.2",
-    "style-loader": "^2.0.0",
-    "webpack": "^5.76.2",
-    "webpack-cli": "^4.10.0"
-  },
+  "name": "starter-files",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "description": "Final code for the abbreviation plugin tutorial part 1 (starter files for part 2).",
   "scripts": {
-    "build": "webpack --mode development"
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
   },
   "devDependencies": {
+    "vite": "^5.2.0",
     "@ckeditor/ckeditor5-inspector": "^4.1.0"
   },
-  "name": "part-1",
-  "version": "1.0.0",
-  "description": "Final code for the abbreviation plugin tutorial part 1 (starter files for part 2).",
-  "main": "app.js",
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com/ckeditor/ckeditor5-tutorials-examples.git"
+  "dependencies": {
+    "ckeditor5": "^41.3.0-alpha.4"
```

Comment:
Leave comment to remember we need replace version with proper one. The same with others.